cli/src/main/scala/scalaxb/compiler/Config.scala

```diff
@@ -85,15 +90,15 @@ object Config {
   val defaultOutdir = Outdir(new File("."))
   val defaultWrappedComplexTypes = WrappedComplexTypes(Nil)
   val defaultProtocolFileName = ProtocolFileName("xmlprotocol.scala")
-  val defaultProtocolPackageName = ProtocolPackageName(None)
+  val defaultProtocolPackageName = "scalaxb.protocol"
```

Comment:
I don't want `scalaxb` package used for the code that we didn't write. The default package should be `"generated"`.